atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -103,10 +103,24 @@ def __init__(self, tasker, workflow, labels=None, dont_overwrite=None,
         # call parent constructor
         super(AddLabelsPlugin, self).__init__(tasker, workflow)
 
-        self.labels = get_image_labels(self.workflow, labels)
+        if isinstance(labels, str):
+            labels = json.loads(labels)
+
+        # see if REACTOR_CONFIG has any labels. If so, merge them with the existing argument
+        # and otherwise use the existing argument
+        image_labels = get_image_labels(self.workflow, False)
+
+        if image_labels:
+            if isinstance(image_labels, str):
```

Comment:
This is coming from REACTOR_CONFIG which wouldn't allow `image_labels` to be of type `str`.

Reply:
Fixed.